components/cli/pkg/commands/list_ingresses.go

```diff
@@ -314,6 +370,33 @@ func displayCellImageApisTable(cellImageContent *image.Cell) {
 	table.Render()
 }
 
+func getIngressValues(cellImageContent string) kubectl.Cell {
+	spinner := util.StartNewSpinner("Extracting Cell Image " + util.Bold(cellImageContent))
```

Comment:
Do we need a spinner here? Also spinner is not stopped at the end of the function.

Reply:
Fixed